lib/hz2600/CallflowBuilder/Node/Offnet.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace CallflowBuilder\Node; 
+
+class Offnet extends AbstractNode
```

Comment:
I'd argue we should not expose this, since it just redirects into the "resources" callflow action with `use_local_resources` set to `false`. You could have two classes, GlobalResource and LocalResource, that extend a base Resource class, each setting the `use_local_resource` flag appropriately, and having all the other properties in the Resource class. Just a thought